src/GraphODataTemplateWriter/CodeHelpers/TypeScript/TypeScriptNamespace.cs

```diff
@@ -36,16 +36,16 @@ public class TypeScriptNamespace
         // constants
         private const int MaxLineLength = 120;
         private const string MainNamespaceName = "Microsoft.Graph";
-        private string TypeScriptMainNamespaceName = "microsoftgraph" + (ConfigurationService.Settings.Properties != null && ConfigurationService.Settings.Properties.ContainsKey("typescript.namespacePostfix") ? ConfigurationService.Settings.Properties["typescript.namespacePostfix"] : string.Empty);
+        private const string TypeScriptMainNamespaceNamePrefix = "microsoftgraph";
         private const string TabSpace = "    ";
 
         /// <summary>
         /// Returns the main or top level namespace
         /// </summary>
         /// <returns></returns>
-        public string GetMainNameSpace()
+        public string GetMainNamespace()
         {
-            return TypeScriptMainNamespaceName;
+            return TypeScriptMainNamespaceNamePrefix + (ConfigurationService.Settings.Properties != null && ConfigurationService.Settings.Properties.ContainsKey("typescript.namespacePostfix") ? ConfigurationService.Settings.Properties["typescript.namespacePostfix"] : string.Empty);
```

Comment:
style: Please consider splitting this into multiple lines. And if you use null-conditional operator:
```suggestion
            var properties = ConfigurationService.Settings?.Properties;
            return properties?.ContainsKey("typescript.namespacePostfix") == true
                ? TypeScriptMainNamespaceNamePrefix + properties["typescript.namespacePostfix"]
                : TypeScriptMainNamespaceNamePrefix;
```